@stellar/design-system-website/src/generated/gitInfo.ts

```diff
@@ -1 +1 @@
-export default { commitHash: "a02b763" };
+export default { commitHash: "d5b7412" };
```

Comment:
this is basically a cache bust, right?

Reply:
Yeah, that file is auto-generated with the repo GitHub hash. It was added to make sure we see the latest website version (was handy when we had the caching issues earlier).

![image](https://user-images.githubusercontent.com/7346473/126537117-8167b526-0e31-4b4a-9724-e8b0460db219.png)